scripts/ci/jobs/ocp_scanner_v4_install_tests.py

```diff
@@ -16,6 +17,20 @@
 os.environ["ROX_SCANNER_V4"] = "true"
 os.environ["ENABLE_OPERATOR_TESTS"] = "true"
 
+try:
+    # SFA Agent supports OCP starting from 4.16, since we test oldest (4.12) and
+    # latest (4.20 at the moment), exclude the former one.
+    # We expect CLUSTER_FLAVOR_VARIANT be the following format:
+    #   openshift-4-ocp/stable-${major_version}.${minor_version}
+    ocp_variant = os.environ.get('CLUSTER_FLAVOR_VARIANT', '')
+    EXPR = r"openshift-4-ocp/\w+-(?P<major>\d+).(?P<minor>\d+)"
+
+    m = re.match(EXPR, ocp_variant)
+    if int(m.group("major")) >= 4 and int(m.group("minor")) >= 16:
+        os.environ["SFA_AGENT"] = "Enabled"
+except Exception as ex:
+    print(f"Could not identify the OCP version, {ex}, SFA is disabled")
+
```

Comment:
There's an almost identical block in `ocp_qa_e2e_tests.py` but there the check is still `int(m.group("minor")) > 16`. You could extract this into a common utils file.

Reply:
Ok, but at least the version check could be in sync.